_sass/_main.sass

```
@@ -161,6 +161,15 @@ $monospaced-font: "Roboto Mono"
 .small
   font-size: $small-size
 
+.float-left
+  float: left;
+
+.float-right
+  float: right;
+
+.medium-image
+  max-width: 50%;
```

Kód super, odstranil bych tady jen středníky

```suggestion
.float-left
  float: left

.float-right
  float: right

.medium-image
  max-width: 50%
```